apps/projects/app/components/Panel/PanelComponents.js

```diff
@@ -40,14 +42,111 @@ export const IssueTitle = ({ issue }) => (
 )
 IssueTitle.propTypes = issueShape
 
-export const IssueText = styled.div`
-  white-space: nowrap;
-  overflow: hidden;
-  text-overflow: ellipsis;
+export const Avatar = ({ user }) => {
+  const theme = useTheme()
+
+  return (
+    <Link
+      href={user.url}
+      target="_blank"
+      style={{ textDecoration: 'none', color: `${theme.link}`, marginRight: 6 }}
```

Comment:
Why not this?

```jsx
css={`text-decoration: none; color: ${theme.link}; margin-right: ${GU}px`}

Reply:
Probably historicaly reasons. Fixed, both places.